packages/manager/apps/vrack-services/src/components/DataGrid.tsx

```diff
@@ -35,103 +75,122 @@ export const Datagrid: React.FC<Props> = ({ data }) => {
           </tr>
-        ))}
-      </tbody>
-    </table>
+        </thead>
+        <tbody>
+          {data.map((currentVs: VrackServices) => (
+            <tr
+              key={`tr-${currentVs.id}`}
+              className="bg-white border-2 border-solid border-ods-primary-100 background-ods-primary-000"
+            >
+              <td className="p-6 text-center">
+                <EditableText
+                  disabled={currentVs.resourceStatus !== ResoureceStatus.READY}
+                  defaultValue={currentVs.currentState.displayName}
+                  onEditSubmitted={async (value) => {
+                    await updateVS({
+                      vrackServicesId: currentVs.id,
+                      checksum: currentVs.checksum,
+                      targetSpec: {
+                        displayName: value || null,
+                        subnets: currentVs.currentState.subnets || [],
+                      },
+                    });
+                    queryClient.invalidateQueries({
+                      queryKey: updateVrackServicesQueryKey({
+                        vrackServicesId: currentVs.id,
+                        checksum: currentVs.checksum,
+                        targetSpec: currentVs.targetSpec,
+                      }),
+                    });
+                  }}
+                >
+                  <OsdsLink
+                    color={ODS_THEME_COLOR_INTENT.primary}
+                    onClick={() => navigate(`/${currentVs.id}`)}
+                  >
+                    {currentVs.currentState.displayName || currentVs.id}
+                  </OsdsLink>
+                </EditableText>
+              </td>
+              <td className="p-6 text-center">
+                <OsdsChip
+                  inline
+                  color={
+                    colorByProductStatus[currentVs.currentState.productStatus]
+                  }
+                >
+                  {t(currentVs.currentState.productStatus)}
+                </OsdsChip>
+              </td>
+              <td className="p-6 text-center">
+                {t(currentVs.currentState.zone)}
+              </td>
+              <td className="p-6 text-center">
+                {currentVs.currentState.vrackId}
+              </td>
+              <td className="p-6 text-center">
+                {currentVs.createdAt
+                  ? new Date(currentVs.createdAt).toLocaleDateString(
+                      locale.replace('_', '-'),
+                    )
+                  : '-'}
+              </td>
+              <td className="p-6 text-center">
+                <OsdsButton
+                  inline
+                  color={ODS_THEME_COLOR_INTENT.error}
+                  variant={ODS_BUTTON_VARIANT.stroked}
+                  type={ODS_BUTTON_TYPE.button}
+                  size={ODS_BUTTON_SIZE.sm}
+                  disabled={isUpdateLoading || undefined}
+                  onClick={() => console.log('delete ', currentVs.id)}
```

Comment:
Console log ?

Reply:
Yes waiting for the process to delete a vrack services
we don't have the specs yet